src/textual/screen.py

```diff
@@ -22,15 +22,16 @@
     from typing_extensions import Final
 
 # Screen updates will be batched so that they don't happen more often than 20 times per second:
-UPDATE_PERIOD: Final = 1 / 20
+UPDATE_PERIOD: Final = 1 / 60
```

Comment:
I'm rewriting an application of mine, moving from an `ncurses` TUI + non-uniform CLI interface to rich+textual in order to attempt a unified UI for the user.
I couldn't help but notice that scrolling through the lines on a table in my old `ncurses` code is significantly smoother than scrolling in `sandbox/table.py`.

I am aware, that textual is very early stage but I wanted to ask whether you're aware of this, if you have plans on how this may be improved in the future, or in general how your experience in terms of performance has been thus far. Do you "frequently" compare to other TUI libraries?

Reply:
It's very smooth here, but I haven't started optimizing.